src/resources/assets/js/mixins/largoContainer.vue

```diff
@@ -519,6 +600,9 @@ export default {
         resetLabelCount() {
             this.fetchedLabelCount = false;
             this.labels = [];
+        },
+        resetFilteringTab() {
+            this.$refs.filteringTab.resetFilters();
```

Comment:
Typically in a Vue application, the tab component would pass the information about changed filters via events to this container, which then updates the state. The state is in turn propagated via props back to the tab component, that can update the UI accordingly. If the filters should be reset, only the container has to update it's state and the tab will update automatically. Have a look at the sorting tab for an example.

All this is done to decouple the component from the parent and vice versa, so you don't have to make a hard call to a child component method as is done here.